services/net/transcription/appsettings.json

```diff
@@ -0,0 +1,54 @@
+{
+  "BaseUrl": "/",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Warning",
+      "Microsoft": "Error",
+      "TNO": "Information"
+    }
+  },
+  "AllowedHosts": "*",
+  "Service": {
+    "MaxFailLimit": 5,
+    "ApiUrl": "http://api:8080",
+    "TimeZone": "UTC",
+    "CapturePath": "/data/capture",
+    "ClipPath": "/data/clip",
```

Comment:
Change to `FilePath` or some other name to reflect this is the location files are stored.

Reply:
Fixed.